File: problemas/iniciante/1534/1534.lua
while true do
  entrada = io.read()
  if entrada == nil then
    break
  end

  n = tonumber(entrada, 10)
  for i = 1, n do
    for j = 1, n do
      if (i - 1 == n - 1 - j + 1) then
        io.write(2)
      else
        if (i == j) then
          io.write(1)
        else
          io.write(3)
        end
      end
    end
    print("")
  end
end

File: problemas/iniciante/1534/1534.clj
(require '[clojure.java.io :as io])

(defn leitura [entrada p]
  (if (< p (count entrada))
    (do
      (let [n (read-string (nth entrada p))]
        (doseq [i (range 0 n)]
          (doseq [j (range 0 n)]
            (if (= i (- n 1 j))
              (printf "2")
              (if (= i j)
                (printf "1")
                (printf "3"))))
          (printf "\n")))
      (leitura entrada (+ p 1)))
    ()))

(let [entrada (vec (line-seq (io/reader *in*)))]
  (leitura entrada 0))